### DISCUSSION TITLE
TSML 3.14.8

### DISCUSSION THREAD

**Discussion by Grateful-Viv** [code4recovery/12-step-meeting-list#1015] [Q&A] [answered]
I did do the update and see that the fix was added so I don't have to go to the meeting list from the admin side to edit a meeting. It is now there on the WP Admin Dashboard.

I also see the option to switch from legacy to beta geocoding is no longer.

But can you tell me what the following means and if I should pay attention?

- Upgrade dev dependencies
- Overriding theme files no longer overrides TSML UI setting
- Remove automatic taxonomy pages
- Add ability for meetings page to be appended to site URL (rather than blog URL)

Thank you.

**Comment by joshreisner** [comment, marked as answer]
uh no not really! we'll announce it if there's a major change that affects lots of users. these were all pretty niche issues which are visible under Issues > Closed and/or Pull Requests > Closed

going forward i'll try linking these changes to the issues they address so you can see more info if you're curious

**Reply by Grateful-Viv** [reply]
Thanks @joshreisner and I appreciate the fix on the admin side to edit a meeting.
